### PR TITLE
Clean up style in non-app files with Black (part of #123)

### PR DESCRIPTION
**Pull request by justwheel**
This commit reformats several miscellanenous Python files/scripts using
the Black linter, added in the `code_lint.sh` script. These changes are
all cosmetic and should not change functionality.

Since @leong96 mentioned he is working on comments, I will hold off on
the app Python files inside of `grasa_event_locator/` since it would
cause messy merge conflicts. I'll circle back to this once @leong96
wraps up any remaining changes to the app files with comments.

Part of #123 (but does not close it).

### CODE REVIEW

File: up.py [hunk 1]
```diff
@@ -7,15 +7,18 @@
...
 parser = argparse.ArgumentParser()
 group = parser.add_mutually_exclusive_group()
-group.add_argument("--setup", help="initial setup", action='store_true')
-group.add_argument("--start", help="start the app", action='store_true')
-group.add_argument("--stop", help="stop the app", action='store_true')
-group.add_argument("--restart", help="Restart the container without rebuilding", action='store_true')
+group.add_argument("--setup", help="initial setup", action="store_true")
+group.add_argument("--start", help="start the app", action="store_true")
+group.add_argument("--stop", help="stop the app", action="store_true")
+group.add_argument(
```
Comment (nlMeminger) [inline up.py:13]: its obviously not perfect, but the extra whitespace added here that breaks with the rest of the code above, just looks odd.

File: up.py [hunk 2]
```diff
@@ -24,76 +27,92 @@ def cmd_run(command):
...
-    results = cmd_run('docker build -t {} .'.format(imageName))
+    print("building container:")
+    results = cmd_run("docker build -t {} .".format(imageName))
     checkResults(results)
-    
-    print('Migrating Database:')
-    results = cmd_run('sudo docker run --rm -it --entrypoint "./manage.py" {} "migrate"'.format(imageName))
+
+    print("Migrating Database:")
+    results = cmd_run(
```
Comment (nlMeminger) [inline up.py:65]: another case of adding whitespce making it somewhat harder to read, in my opinion

**Review by nlMeminger — CHANGES_REQUESTED** [review]
Most of the formatting changes make things a lot cleaner to look at, but there are a couple places where adding new lines and whitespace make it a little harder to look at

**Comment by justwheel** [comment]
Thanks for reviewing @nlMeminger. The issue with this change is these are all personal preferences. For example, I like some of the syntax above because it enforces specific line lengths, which makes things like git diffs easier to read.

Because these formatting decisions are so personal and subjective, I decided to use [Black](https://github.com/psf/black) and its specifications based on the [standardized PEP-8 style guide](https://pep8.org) for Python code. This way, we don't have to keep track of the style rules ourselves: we use a standard tool used in the Python developer community, developed by the Python Software Foundation.

I respect your feedback on these changes, but I waited until the end of the project to do these style enforcement PRs because of how it drastically changes the appearance of the code. I knew not everyone would agree. However, I want to stick as much as possible to an automated way of doing this and better unify our different coding styles together.

**Review by nlMeminger — APPROVED** [review]
good enough for me

**Pull request merged**